db/schema.sql

```diff
@@ -0,0 +1,13 @@
+DROP TABLE IF EXISTS users;
+
+CREATE TABLE users (
+    id SERIAL PRIMARY KEY,
+    age integer,
+    name varchar(255),
+   company varchar(255),
+    country varchar(255),
+    photo text,
+    about varchar(255),
```

Comment:
I think varchar(255) might be too restrictive for the about field. It's a whole paragraph so it could easily be longer than 255 characters. I would make this `text`